Testscripts/Windows/KERNEL-DEBUG-BASIC.ps1

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+    This script tests the debug kernel basic functionality.
+
+.Description
+    The script will install the debug kernel and boot into debug kernel,
+    then check whether have call trace
+#>
+param([String] $TestParams,
+      [object] $AllVmData)
+
+function Main {
+    param (
+        $VMname,
+        $HvServer,
+        $Ipv4,
+        $VMPort,
+        $VMUserName,
+        $VMPassword,
+        $RootDir,
+        $TestParams
+)
+
+    if ( ( $global:detectedDistro -imatch "CENTOS") -or ($global:detectedDistro -imatch "REDHAT") ) {
+        Write-LogInfo "Test on DISTRO $global:detectedDistro"
+    }
+    else {
+        Write-LogInfo "Do not support this DISTRO $global:detectedDistro."
+        return "SKIPPED"
+    }
+    $cmd = ". utils.sh && install_package 'kernel-debug' && grub2-set-default 0"
+    $null = Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 `
+        -port $VMPort -command $cmd -runAsSudo -runMaxAllowedTime 480
+
+    # Rebooting the VM in order to apply the kdump settings
+    Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort `
```

Comment:
Line 38 ~ 44 -> Replace with $TestProvider.RestartAllDeployments($AllVMData)

Reply:
Updated, thank you so much.